cmd/common-main.go

```diff
@@ -179,7 +179,14 @@ func minioConfigToConsoleFeatures() {
 	}
 	// pass the console subpath configuration
 	if value := env.Get(config.EnvMinIOBrowserRedirectURL, ""); value != "" {
-		os.Setenv("CONSOLE_SUBPATH", pathJoin(globalBrowserRedirectURL.Path, SlashSeparator))
+		subPath := ""
+		trimmedRedirectPath := strings.TrimSpace(globalBrowserRedirectURL.Path)
+		if trimmedRedirectPath != "" && trimmedRedirectPath != SlashSeparator {
+			subPath = pathJoin(trimmedRedirectPath, SlashSeparator)
+		}
+		if subPath != SlashSeparator {
+			os.Setenv("CONSOLE_SUBPATH", subPath)
+		}
```

Comment:
Okay this is wrong you should just do `path.Clean(pathJoin(globalBrowserRedirectURL.Path, SlashSeparator))`

Reply:
done, that also worked